src/main/scala/nl.knaw.dans.easy.deposit/docs/DatasetXml.scala

```diff
@@ -0,0 +1,132 @@
+    <ddm:DDM
+      xmlns:dc="http://purl.org/dc/elements/1.1/"
+      xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+      xmlns:dcterms="http://purl.org/dc/terms/"
+      xmlns:dcx-dai="http://easy.dans.knaw.nl/schemas/dcx/dai/"
+      xmlns:ddm="http://easy.dans.knaw.nl/schemas/md/ddm/"
+      xsi:schemaLocation="http://easy.dans.knaw.nl/schemas/md/ddm/ http://easy.dans.knaw.nl/schemas/md/2017/09/ddm.xsd"
+    >
+      <ddm:profile>
+        { requiredElems(dm.titles, "dcterms:title").addAttr(lang) }
+        { elems(dm.alternativeTitles, "dcterms:alternative").addAttr(lang) }
+        { requiredElems(dm.descriptions, "dc:description").addAttr(lang) }
+        { requiredElems(dm.creators, "dcx-dai:creatorDetails", lang) }
+        { requiredElems(dm.dates.map(filter(_, Seq(created))), "ddm:created") }
+        { requiredElems(dm.dates.map(filter(_, Seq(available))), "ddm:available") }
+        { requiredElems(dm.audiences, "ddm:audience") }
+        { requiredElems(dm.accessRights.map(Seq(_)), "ddm:accessRights") }
+      </ddm:profile>
+      <ddm:dcmiMetadata>
+        { elems(dm.contributors, "dcx-dai:creatorDetails", lang) }
+        { elems(dm.publishers, "dcterms:publisher").addAttr(lang) }
+        { elems(dm.sources, "dc:source").addAttr(lang) }
+        { elems(dm.dates.map(filter(_, otherDates)), qualifier) }
+        { optionalElem(dm.license, "dcterms:license") /* TODO xsi:type="dcterms:URI" not supported by json */ }
+      </ddm:dcmiMetadata>
+      <ddm:additional-xml>
+      </ddm:additional-xml>
+    </ddm:DDM>
+  }
+
+  // called by requiredElems, elems, optionalElem
+  private def elem[T](target: String, lang: MaybeAttribute)(source: T): Elem = (source match {
+    case a: Author => <key>{authorDetails(a, lang)}</key>
+    case a: AccessRights => <key>{a.category.toString}</key>
+    case SchemedKeyValue(scheme, key, _) => <key scheme={scheme.toString}>{key}</key> // e.g. role, audience
+    case QualifiedSchemedValue(None, value, _) => <key>{value}</key>
+    case QualifiedSchemedValue(Some(scheme), value, _) => <key scheme={scheme.toString}>{value}</key>
+    case PossiblySchemedKeyValue(Some(scheme), key, _) => <key scheme={scheme.toString}>{key}</key>
+    case PossiblySchemedKeyValue(None, key, _) => <key>{key}</key>
+    case v => <key>{v}</key>
+  }).copy(label = elemLabel(target, source))
+
+  /**
+   * @param target the default label
+   * @param source may have a qualifier or some other property that determines the label
+   */
+  private def elemLabel[T](target: String, source: T) = {
+    val rightsholder = "rightsholder"
+    source match {
+      case x: QualifiedSchemedValue[_, _] if target == qualifier => x.qualifier.toString
+      case Author(_, _, _, _, Some(SchemedKeyValue(_, _, `rightsholder`)), _, _) => "dcterms:rightsholder"
+      case _ => target
+    }
+  }
+
+  private def authorDetails(author: Author, lang: MaybeAttribute) =
```

Comment:
when the author is an organization and it has a role, the output would be like:

https://github.com/DANS-KNAW/easy-split-multi-deposit/blob/ea7c2dc3d6a78cacd779d2efcb2d799e399e22cf/src/main/scala/nl.knaw.dans.easy.multideposit/actions/AddDatasetMetadataToDeposit.scala#L88-L89